Orm/Xtensive.Orm.Manual/Prefetch/PrefetchTest.cs

```diff
@@ -250,7 +249,7 @@ public async Task MultipleBatchesAsyncTest()
         transactionScope.Complete();
       }
 
-      using (var session = domain.OpenSession())
+      await using (var session = await domain.OpenSessionAsync())

```

Comment:
```suggestion
      var session = await domain.OpenSessionAsync().ConfigureAwait(false);
      await using (session.ConfigureAwait(false))
```

Reply:
There is o need to do this in test code in case we don't synchronously wait for async method completion.